# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1245127436 -7200
#      Tue Jun 16 06:43:56 2009 +0200
# Branch release-3-2-x
# Node ID 1ffb54887c425efc5ae3d616ed940fd038999a6a
# Parent  f944142010ced613089896914743151ddfcf8d45
quiver.m: Add 'clf' to demos.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,10 +1,11 @@
 2009-06-14  Ben Abbott <bpabbott@mac.com>
 
+	* plot/quiver.m: Add 'clf' to demos.
 	* plot/plotyy.m: Fix compatibility with subplot, add listeners for
 	dataaspectratio, and add a demo.
 
 2009-06-14  Eric Chassande-Mottin <echassandemottin@gmail.com>
 
 	* plot/plotyy.m: Correct behavior when there is no currentfigure.
 
 2009-06-11  Ben Abbott <bpabbott@mac.com>
diff --git a/scripts/plot/quiver.m b/scripts/plot/quiver.m
--- a/scripts/plot/quiver.m
+++ b/scripts/plot/quiver.m
@@ -77,16 +77,17 @@ function retval = quiver (varargin)
 
   if (nargout > 0)
     retval = tmp;
   endif
 
 endfunction
 
 %!demo
+%! clf
 %! [x,y] = meshgrid(1:2:20);
 %! h = quiver(x,y,sin(2*pi*x/10),sin(2*pi*y/10));
 %! set (h, "maxheadsize", 0.33);
 
 %!demo
 %! axis("equal");
 %! x=linspace(0,3,80); y=sin(2*pi*x); theta=2*pi*x+pi/2;
 %! quiver(x,y,sin(theta)/10,cos(theta)/10);
